lib/nerves_ssh/options.ex

```diff
@@ -228,6 +231,9 @@ defmodule NervesSSH.Options do
   defp shell_opts(%{shell: :lfe}), do: [{:shell, {:lfe_shell, :start, []}}]
   defp shell_opts(%{shell: :disabled}), do: [shell: :disabled]
 
+  defp cli_opts(%{cli: nil}), do: []
+  defp cli_opts(%{cli: cli}), do: [ssh_cli: cli]
```

Comment:
I think that I would vote we leave the options.ex file alone for now. If a custom shell is being used, we just supply the options in `daemon_option_overrides: [ssh_cli: opts]`